website/docs/reference/node-selection/state-comparison-caveats.md

```diff
@@ -60,7 +53,7 @@ dbt will do its best to capture *only* changes that are the result of modificati
 - iterative improvements to dbt's built-in detective abilities
 - better options for more complex projects, in the form of more-specific sub-selectors (see [this issue](https://github.com/dbt-labs/dbt-core/issues/2704))
 
-State comparison is now able to detect env-aware config in `dbt_project.yml`. For instance, this target-based config would register as a modification in v0.18.0, but in v0.19.0 it no longer will:
```

Comment:
We can probably remove lines 52-54 at this point. The issue is closed with a linked Core PR. Thoughts?

Reply:
@matthewshaver great idea and thanks for that context. Done!